core/pom.xml

```diff
@@ -76,6 +76,18 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.apache.kafka</groupId>
```

Comment:
No, tests won't compile without this dependency. It's needed for the use of `TestSslUtils`.